plugins/meta/portmap/portmap.go

```diff
@@ -249,7 +249,7 @@ func fillDnatRules(c *chain, config *PortMapConf, containerIP net.IP) {
 			copy(hpRule, ruleBase)
 
 			hpRule = append(hpRule,
-				"-s", containerIP.String(),
+				"-s", containerNet.String(),
```

Comment:
@AlbanBedel Brian and I think it's unclear and wrong to pass the unmasked IP (eg 172.30.16.2/24) to iptables, since iptables will simply mask off the host bits and actually use 172.30.16.0/24. So instead here on line 252, could you do something ilke:

```
ipnet := net.IPNet{IP: containerNet.IP.Mask(containerNet.Mask), Mask: containerNet.Mask}
hpRule = append(hpRule,
        "-s", ipnet.String(),
```

And then update the readme example iptables rules to use the network (172.16.30.0/24) instead.

Reply:
It might be unusual for some but it is definitely not wrong. It is not luck or due to some implementation quirk that iptables mask the low bits, that's exactly how a network address must be interpreted. What you suggest just add useless code.

Reply:
FWIW, I ran this PR locally and confirmed that iptables rules show just the prefix/mask (i.e. host bits are all 0) in the iptables chain.  So the rules show e.g: `192.168.0.0/16` and not `192.168.0.2/16`

Reply:
Then I'll fix the README to match what iptables report for the rules we add.

Reply:
Yes it's valid and I think clarity is not a useful thing, but I'm OK with this.